airbyte-integrations/connectors/source-mysql/build.gradle

```diff
@@ -35,8 +40,37 @@ dependencies {
     performanceTestJavaImplementation project(':airbyte-integrations:bases:standard-source-test')
     performanceTestJavaImplementation project(':airbyte-integrations:connectors:source-mysql')
 
+    // TODO: remove this once the code has migrated to the CDK
     implementation files(project(':airbyte-integrations:bases:base-java').airbyteDocker.outputs)
     integrationTestJavaImplementation files(project(':airbyte-integrations:bases:base-java').airbyteDocker.outputs)
     performanceTestJavaImplementation files(project(':airbyte-integrations:bases:base-java').airbyteDocker.outputs)
 }
 
+task buildCDK(type: Exec) {
```

Comment:
A pending change will move this to the root build.gradle so it is available for all connectors. Basically the purpose of this task is to build the CDK and publish locally _if_ the connector depends on the latest snapshot build of the CDK - meaning that the CDK and connector are being co-developed in the same PR.

If using a pinned non-snapshot version of the CDK, then this build task is a no-op.